document_parameters.rb

```diff
@@ -1,33 +1,44 @@
-require_relative 'rosette_api_error'
+require_relative 'bad_request_format_error'
 
 class DocumentParameters
   attr_accessor :content, :content_uri, :file_path, :language
 
-  def initialize(content=nil, content_uri=nil, file_path=nil, language=nil)
-    @content = content
-    @content_uri = content_uri
-    @file_path = file_path
-    @language = language
+  def initialize(options = {})
```

Comment:
When there are a bunch of parameters using an `options` hash is frequently preferred.